<launch>
	<node name="render" pkg="mesh_localize" type="render_node" output="screen">
		<param name="step_size" type="double" value="0.3"/>
	</node>
        <!-- Start mesh_localize_node -->
	<node name="mesh_localize" pkg="mesh_localize" type="mesh_localize_node" output="screen" >
                <remap from="image" to="/virtual_camera/image" />
		<remap from="camera_info" to="/virtual_camera/camera_info" />

		<param name="enable_edge_tracking" type="bool" value="true"/>
		<param name="show_debug" type="bool" value="true"/>
		<param name="show_pnp_matches" type="bool" value="true"/>
		<param name="show_global_matches" type="bool" value="true"/>
		<param name="virtual_image_source" type="string" value="ogre"/>
		<param name="pnp_descriptor_type" type="string" value="orb"/>
		<param name="img_match_descriptor_type" type="string" value="surf"/>
		<param name="global_localization_alg" type="string" value="depth_feature_match"/>
		<param name="ogre_data_dir" type="string" value="$(find mesh_localize)/data/box"/>
		<param name="ogre_cfg_dir" type="string" value="$(find mesh_localize)/ogre_cfg/"/>
		<param name="ogre_model" type="string" value="box.mesh"/>
		<param name="image_scale" type="double" value="1.0"/>
	</node>
</launch>
